Kickstarter-iOS/DataSources/PledgeDataSource.swift

```diff
@@ -10,26 +10,80 @@ final class PledgeDataSource: ValueCellDataSource {
     case summary
   }
 
-  func load(amount: Double, currency: String, delivery: String, isLoggedIn: Bool) {
+  enum PledgeInputRow {
+    case pledgeAmount(amount: Double, currency: String)
+    case shippingLocation(location: String, amount: Double, currencyCode: String)
+
+    var isShipping: Bool {
+      switch self {
+      case .shippingLocation: return true
```

Comment:
I thought we could just do `return self == .shippingLocation` 🤔 but I see the compiler isn't happy doing that.